### ISSUE TITLE
NPE when android and java project included into Build.kt

### ISSUE THREAD

**Issue by fboldog** [cbeust/kobalt#43]
- Kobalt: 0.283
- JDK: 1.8.0_51

```
Exception in thread "main" kotlin.KotlinNullPointerException
	at com.beust.kobalt.internal.TaskManager.calculateTransitiveClosure(TaskManager.kt:177)
	at com.beust.kobalt.internal.TaskManager.access$calculateTransitiveClosure(TaskManager.kt:13)
	at com.beust.kobalt.internal.TaskManager.runTargets(TaskManager.kt:91)
	at com.beust.kobalt.Main.runWithArgs(Main.kt:196)
	at com.beust.kobalt.Main.access$runWithArgs(Main.kt:46)
	at com.beust.kobalt.Main$run$seconds$1.invoke(Main.kt:97)
	at com.beust.kobalt.Main$run$seconds$1.invoke(Main.kt:46)
	at com.beust.kobalt.misc.BenchmarksKt.benchmark(Benchmarks.kt:5)
	at com.beust.kobalt.Main.run(Main.kt:95)
	at com.beust.kobalt.MainKt.mainNoExit(Main.kt:43)
	at com.beust.kobalt.MainKt.main(Main.kt:25)
```

**Comment by cbeust** [comment]
Can you include `--log 2` and your `Build.kt`?

**Comment by fboldog** [comment]
[kobalt284-log2-dev.txt](https://github.com/cbeust/kobalt/files/40664/kobalt284-log2-dev.txt)

**Comment by fboldog** [comment]
[Build.kt.txt](https://github.com/cbeust/kobalt/files/40666/Build.kt.txt)

**Comment by fboldog** [comment]
Fixed in: 0.285
